helm/drone-fork-approval-extension/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "drone-fork-approval-extension.fullname" . }}
+  labels:
+    {{- include "drone-fork-approval-extension.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "drone-fork-approval-extension.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        checksum/secret: {{ .Values.secret | sha256sum }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "drone-fork-approval-extension.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
```

Comment:
probably want to define this not what helm allows / except as config. So run as non root, disallow privilege escalation and drop all capabilities (this would also require changing the default bind off of port 80).

Reply:
💯 , we don't need to listen on 80.

There's an example `securityContext` that we use for `teleport-kube-agent` here: https://github.com/gravitational/teleport/blob/925801440a3eb8b0431c4a4e3245f2050fd04bad/examples/chart/teleport-kube-agent/templates/deployment.yaml#L47-L54

Reply:
Port changed to 3888 in 3a19c7e.
Hardening applied in 92336c8.